include/OpenSpaceToolkit/Astrodynamics/Trajectory/State/CoordinatesBroker.hpp

```diff
@@ -132,16 +132,6 @@ class CoordinatesBroker
     VectorXd extract(const VectorXd& allCoordinates, const Shared<const CoordinatesSubset>& aCoordinatesSubsetSPtr)
         const;
 
-    /// @brief              Create a shared pointer wiht the given coordinates subsets.
-    ///
-    /// @param              [in] aCoordinatesSubsetsArray the coordinates subsets to consider
-    ///
-    /// @return             A new instance
-
-    static Shared<const CoordinatesBroker> FromSubsets(
```

Comment:
This method is redundant as it can be replicated with a single line:
`std::make_shared<CoordinateBroker>(aCoordinateSubsetsArray);`